malayalam_morpheme_splitter/install.py

```diff
@@ -0,0 +1,22 @@
+import os
+import shutil
+import pkg_resources
+
+def main():
+   
+    target_dir = os.getcwd()
```

Comment:
Are you saving in current directory? Will that work?

Scenario: User installs the library globally uisng pip install from a folder A. Now he goes to some other project in folder B and uses the smae library installed globally. Will he have access to the data in folder A?